Kodilla-testing/src/main/java/com/kodilla/testing/shape/ShapeCollector.java

```diff
@@ -5,21 +5,36 @@
 
 public class ShapeCollector {
 
-    List<Shape> figureList = new ArrayList<>();
+    List<Shape> figuresList = new ArrayList<>();
```

Comment:
Co to?

Reply:
No właśnie IH wysyła też to, czego nie chcę, mimo, że katuje go komendą 'git status' albo dodaje osobno projekty przez 'git add <file>'